sw/device/tests/consecutive_irqs/consecutive_irqs_test.c

```diff
@@ -13,10 +13,12 @@
 #include "sw/device/lib/runtime/hart.h"
 #include "sw/device/lib/runtime/ibex.h"
 
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
```

Comment:
Nit: this is missing a `// Generated.` comment.

Reply:
I wasn't sure either way on this. The `*_regs.h` is generated by the build system, so there the comment is useful because you won't find this header in the repository.

With `top_earlgrey.h`, the header is generated, but it's also committed to the repo, so you will find it when grepping for what's in it. I can see the value either way, and I'm happy to add the comment, but it would be useful to know where that split should lie.

Reply:
I see your point, I guess you could generate a comment in the `hw/top_earlgrey/sw/autogen/top_earlgrey.h` itself, saying - "I am generated ...".

Reply:
Ok, I will follow-up with replacing the comments in the template. For the moment, I'll add `// Generated.` to this line.